PhilipFunkos/src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import "./App.css";
+import Home from "./home";
+import Product from "./products";
+import Contact from "./contact";
+
+function App() {
+  function home() {
+    let page1 = document.querySelector(".home-page");
+    let page2 = document.querySelector(".products-page");
+    let page3 = document.querySelector(".contact-page");
+    page1.classList.remove("disable");
+    page2.classList.add("disable");
+    page3.classList.add("disable");
+  }
+  function product() {
+    let page1 = document.querySelector(".home-page");
+    let page2 = document.querySelector(".products-page");
+    let page3 = document.querySelector(".contact-page");
+    page2.classList.remove("disable");
+    page1.classList.add("disable");
+    page3.classList.add("disable");
+  }
+  function contact() {
```

Comment:
You can control component visibility in react instead, this code is very messy.

You can create state that holds data on which page to show, and only render this page. Change that state on click.